tensorflow/python/keras/activations.py

```diff
@@ -68,19 +68,50 @@ def softmax(x, axis=-1):
 
 @keras_export('keras.activations.elu')
 def elu(x, alpha=1.0):
-  """Exponential linear unit.
+  """Exponential Linear Unit.
+  The exponential linear unit (ELU) with `alpha` > 0 is:
+  `x` if `x > 0` and `alpha * (exp(x)-1)` if `x < 0`
+  The ELU hyperparameter `alpha` (α) controls the value to which an
+  ELU saturates for negative net inputs. ELUs diminish the
+  vanishing gradient effect.
+
+  ELUs have negative values which pushes the mean of the activations
+  closer to zero.
+  Mean activations that are closer to zero enable faster learning as they
+  bring the gradient closer to the natural gradient.
+  ELUs saturate to a negative value when the argument gets smaller.
+  Saturation means a small derivative which decreases the variation
+  and the information that is propagated to the next layer.
+
+  ![](https://akiraaptx.files.wordpress.com/2017/01/screen-shot-2017-01-14-at-1-39-39-pm.png)
```

Comment:
Very informative graphic, good work @Bharat123rox

Reply:
I've done something similar for SELU in #27837, if you could review that for formatting as well...thanks!!

Reply:
Because this image is hosted by a third-party, we cannot it include it in the docstring. It would need to be hosted on tensorflow.org.